qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -47,6 +47,7 @@
 from qiskit.transpiler.passes import ASAPSchedule
 from qiskit.transpiler.passes import AlignMeasures
 from qiskit.transpiler.passes import ValidatePulseGates
+from qiskit.transpiler.passes import PulseGates
```

Comment:
Is this something that we want to have in the optimization levels? Perhaps #6403 might also be relevant here.

Reply:
`FullPassManager` would be great to manage this new pass. I think we can later switch to this framework.